package.json

```diff
@@ -11,6 +11,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "basic-auth": "^2.0.1",
```

Comment:
Is this package using in somewhere? If not, it should be removed.

Reply:
Did not use this package.
It has been deleted.